CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- ignore flag for commit command
```

Comment:
import the description
something like add `ignore-missing-dependencies` flag to commit command